docs/how-tos/integrating-with-webflow.md

```diff
@@ -0,0 +1,103 @@
+
+![Setting up Webflow element IDs](/img/how-tos/integrating-with-webflow/setting-up-webflow-ids.png)
+
+Next, add the following code to your [custom code at the site level](https://university.webflow.com/lesson/custom-code-in-the-head-and-body-tags?topics=site-settings) in the footer:
+
+:::note
+
+By making sure our elements are “blank” to start with, we can completely reduce any flicker that would be caused by re-writing and element with Javascript. Make sure that you build in logic for default experience in the case that Eppo serves a `null` variation.
+
+:::
+
+```html
+<script>console.log('Loading eppo SDK')</script>
+<script src="https://cdn.jsdelivr.net/npm/@eppo/js-client-sdk@1.4.0/dist/eppo-sdk.js"></script>
+<script>console.log('Done loading eppo SDK')</script>
+
+<script>  
+// Init user parameters
+urlParams = new URLSearchParams(window.location.search);
+subject = urlParams.get('subject') || crypto.randomUUID();
+
+// Prep post-assignment code
+function setHeader() {
+  
+  // Get variation
+  const variationData = window.eppo.getInstance().getParsedJSONAssignment(subject, '<FEATURE-FLAG-KEY>');
+  console.log('>>> got variation data', variationData);
+  
+  // Add client side analytics tracking call to capture experiment data 
+  // and make sure assignment data is sent to your 
+  // data warehouse that is connected to Eppo.
+
+  document.getElementById('header-title').innerText = variationData.title;
+  document.getElementById('header-subtitle').innerText = variationData.subtitle;
+  document.getElementById('header-cta').querySelector('div').innerText = variationData.cta;
+  
+  const headerSrc = variationData.image;
+  
+  const headerImageDiv = document.getElementById('header-image');
+  headerImageDiv.style.backgroundImage = 'url('+headerSrc+')';
+  headerImageDiv.style.backgroundPosition = 'center center';
+  headerImageDiv.style.backgroundRepeat = 'no-repeat';
+  headerImageDiv.style.backgroundSize = 'auto 100%';
+}
+
+// Init Eppo Client
+opts = {apiKey: '<EPPO-SDK-KEY>'};
+window.eppo.init(opts).then(setHeader);
+window.eppo.getInstance().setIsGracefulFailureMode(false);
+</script>
+```
+
+- Provide your SDK key and Feature Flag key in the `'<SDK-KEY>'` and `'<FEATURE-FLAG-KEY>'` placeholders above.
+- Add your client side analytics tracking call once the assignment has been made. Make sure your analytics platform is sending data to your data warehouse connected to Eppo. This will ensure that assignments made by Eppo will be tracked and can be used for experiment analysis.
```

Comment:
Maybe nice to add a little bit more detail here

Reply:
Detail on what specifically? Maybe link to our docs on connecting to an analytics platform?

Reply:
More detail (maybe an example) on how to add the client side analytics tracking call -- I imagine many folks reading this might not immediately know how to do it and it's kinda important to add :)